tools/ci_build/github/linux/docker/build_scripts/finalize.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
```

Comment:
Can you please add a line or two at the top of this and other new files stating what it does? Or may be rename the file appropriately? finalize.sh seems a bit too generic.

Reply:
They are not our code. They are copied from https://github.com/pypa/manylinux .